.gitignore

```diff
@@ -1,5 +1,9 @@
 .pio
 .vscode
+.vs
+*.sln
+*.vcxproj*
+x64
```

Comment:
I saw this files. are you using did you tried on x64 platform using visual studio?

Reply:
I used vs to write the majority of the code. x64 was created when i tried to set it up for platformio. These are just remaining artefacts.